spec/mailers/review_alert_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe ReviewAlertMailer, type: :mailer do
+  ActionMailer::Base.deliveries = []
```

Comment:
It's weird to have this hanging out. If it's required for your test set up (I question if it is though), put it in a before block